### PR TITLE
Integrate data

### PR DESCRIPTION
**Pull request by stg101**
- Rename bad named variables
- Integrate ceoName and companyName to Chart View

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,19 +14,19 @@ function App() {
...
   }
 
-  const [nameCeo, setNameCeo] = React.useState(localStorage.getItem("nameCeo"));
+  const [ceoName, setCeoName] = React.useState(localStorage.getItem("ceoName"));
 
-  function handleNameCeo(value) {
-    localStorage.setItem("nameCeo", value);
-    setNameCeo(value);
+  function handleCeoName(value) {
+    localStorage.setItem("ceoName", value);
```
Comment (test0n3) [inline src/index.js]: This code repeats each time we save to `localStorage`, we have to "streamline" the app later though.

File: src/views/__snapshots__/ceo-view.test.js.snap [hunk 2]
```diff
@@ -2,12 +2,17 @@
...
-      <form>
+      <form
+        class="css-xctk3t-InputNameCeo"
+      >
         <input
+          class="css-lrjsfw-InputNameCeo"
           name="ceo"
-          placeholder="Ceo's name..."
+          placeholder="Ceo's Name"
           type="text"
```
Comment (test0n3) [inline src/views/__snapshots__/ceo-view.test.js.snap]: This `input`needs a `required` property. We cannot allow the user to bypass it or leave it empty. Or at least set a default name.

File: src/views/chart.js [hunk 3]
```diff
@@ -3,82 +3,87 @@ import React from "react";
...
+    id: 7,
+    name: "xyz",
+    name: "Cristian Granda",
+    position: "Tech Lead",
+    parent: 2,
+    children: []
+  },
+  {
+    id: 8,
+    name: "xyz",
```
Comment (test0n3) [inline src/views/chart.js]: Why does it have two names?

File: src/views/__snapshots__/ceo-view.test.js.snap (hunk 2 above)
Comment (stg101) [reply src/views/__snapshots__/ceo-view.test.js.snap]: That is a great point, I am going to create an issue with all this refactoring things.

**Pull request merged**